Catalog/register.json

```diff
@@ -43,5 +43,10 @@
         "org": "PSLmodels",
         "repo": "microdf",
         "branch": "master"
+    },
+    {
+        "org": "PSLmodels",
+        "repo": "OpenFisca-UK",
```

Comment:
@nikhilwoodruff I'm not sure if this matters, but the repo name is all lower-case

Reply:
Thanks @Peter-Metz - just corrected the repo name to OpenFisca-UK